### PR TITLE
packit: disable Centos Stream/fedora ELN teasks

### PR DESCRIPTION
**Pull request by Luap99**
The go version there is only go 1.22.5 but we need go 1.22.6 as of https://github.com/containers/podman/pull/24054

It is not clear to me how to best monitor the repos there to see when they get the update. And then there is the fear that podman keeps updating faster then these envs which makes testing there immposible[1]

[1] https://github.com/containers/image/pull/2550#discussion_r1743588062

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by Luap99** [comment]
@lsm5 PTAL, not sure if this is a good idea (especially the propose downstream for cs10) but I rather not see the failed tests on every PR.

**Review by lsm5 — APPROVED** [review]
I'm told the golang version on centos stream issue should be resolved soon. And this is going into main so shouldn't be a problem.

I'm still looking into how best to monitor repos. copr cli doesn't have anything built in. I think `pkgdb-cli` or `koji/bodhi` could be useful, but we can leave that for later.

/lgtm
/approve
/hold

**Comment by martinpitt** [comment]
@lsm5 https://composes.stream.centos.org/stream-10/development/latest-CentOS-Stream/compose/AppStream/x86_64/os/Packages/ has golang 1.23.1, so I suppose this could be re-enabled? I'm asking because someone from the podman team asked us to enable c-podman testing for podman PRs on CentOS. Happy to do that, but we first need podman builds for that.

**Comment by lsm5** [comment]
> @lsm5 https://composes.stream.centos.org/stream-10/development/latest-CentOS-Stream/compose/AppStream/x86_64/os/Packages/ has golang 1.23.1, so I suppose this could be re-enabled? 

I'll look into that.

> I'm asking because someone from the podman team asked us to enable c-podman testing for podman PRs on CentOS. 

That someone was me :) . Thanks for taking this up. I'll let you know once we have green builds.

**Pull request merged**
